### ISSUE TITLE
can we access to the pretrained model

### ISSUE THREAD

**Issue by yang-fei** [figitaki/deep-retrieval#1]
there is no pretrained model
Can you post it out

**Comment by SergazyK** [comment]
Dataset and pre-trained model posted here: http://www.europe.naverlabs.com/Research/Computer-Vision/Learning-Visual-Representations/Deep-Image-Retrieval .  Link was broken, authors fixed it recently.

**Comment by beshining** [comment]
> Dataset and pre-trained model posted here: http://www.europe.naverlabs.com/Research/Computer-Vision/Learning-Visual-Representations/Deep-Image-Retrieval . Link was broken, authors fixed it recently.

Thank you! I have downloaded the model and prototxt. I simply use the "model.caffemodel" and "deploy_resnet101_normpython.prototxt" to construct a network as following:

caffe_prototxt = './deploy_resnet101_normpython.prototxt'
caffe_weight = './model.caffemodel'
caffe.set_mode_cpu()
net = caffe.Net(caffe_prototxt, caffe_weight, caffe.TEST)
#####

However, the network cannot be constructed (My caffe was not compiled with the normalization layer, so I used normpython.prototxt). Did I use the wrong prototxt or something else go wrong? Errors are listed bellow:

Error parsing text-format caffe.NetParameter: 4539:22: Message type "caffe.LayerParameter" has no field named "roi_pooling_param".
F1225 08:07:01.111474  1430 upgrade_proto.cpp:90] Check failed: ReadProtoFromTextFile(param_file, param) Failed to parse NetParameter file: ./deploy_resnet101_normpython.prototxt

Thank you!

**Comment by DHNicoles** [comment]
@beshining You may have to download and compile the caffe version of Faster-RCNN first.

**Comment by beshining** [comment]
> @beshining You may have to download and compile the caffe version of Faster-RCNN first.

Thank you! I will try it. Hope the problem can be solved

**Comment by rubmarsan** [comment]
Link seems to be broken again (http://download.europe.naverlabs.com/Computer-Vision-CodeandModels/)
Can you guys please fix it?
